app/javascript/app/providers/ghg-emissions-provider/ghg-emissions-provider.js

```diff
@@ -0,0 +1,25 @@
+import { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import * as actions from './ghg-emissions-provider-actions';
+import reducers, { initialState } from './ghg-emissions-provider-reducers';
+
+class LoginProvider extends PureComponent {
```

Comment:
change the name please from `LoginProvider` to `GHGEmissionsProvider` or sth similar, the naming must have stayed here by accident